code/scripting/api/objs/techroom.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include "menuui/readyroom.h"
+#include "cutscene/cutscenes.h"
+#include "scripting/ade_api.h"
+
+namespace scripting {
+namespace api {
+
+DECLARE_ADE_OBJ(l_TechRoomMission, sim_mission);
+DECLARE_ADE_OBJ(l_TechRoomCutscene, cutscene_info);
+
```

Comment:
Similar to my comment in #4602, these should just store an int, the index into their respective arrays.
This is much better, especially if this API should ever be able to be used by _anything_ other than SCPUI

Reply:
Done.